core/driver/sources/src/test/scala/org/virtuslab/ideprobe/dependencies/SingleRunFixtureTest.scala

```diff
@@ -56,8 +56,9 @@ final class SingleRunFixtureTest extends IdeProbeFixture with WorkspaceFixture w
 
   @Test
   def removesDirectoriesEvenAfterFailureToRunIntelliJ(): Unit = {
+    val intellijLauncher = if (OS.Current == OS.Mac) "idea" else "idea.sh"
```

Comment:
Out of curiosity, what's the type of this `idea` file on Mac (_type_ in the sense of e.g. [file command](https://www.tutorialspoint.com/unix_commands/file.htm)

Reply:
Let me paste the output of the `file` command:
```
lkontowski@lkontowski-MacBook-Pro /Applications/IntelliJ IDEA CE.app/Contents/MacOS 
% file idea
idea: Mach-O universal binary with 2 architectures: [x86_64:Mach-O 64-bit executable x86_64
- Mach-O 64-bit executable x86_64] [arm64:Mach-O 64-bit executable arm64
- Mach-O 64-bit executable arm64]
idea (for architecture x86_64):	Mach-O 64-bit executable x86_64
idea (for architecture arm64):	Mach-O 64-bit executable arm64
```

Reply:
Okay... so not a script (`idea.sh`), as on Linux

Reply:
it is an objective c app that reads some stuff from plist, modifies vmoptions and starts IJ